Tita-Clone/Tita-Clone/Resources/SceneDelegate.swift

```diff
@@ -19,7 +19,7 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
             
         let navigationController = UINavigationController(rootViewController: mainViewController)
         navigationController.navigationBar.isHidden = true
-        window?.rootViewController = navigationController
+        window?.rootViewController = LoginViewController()
```

Comment:
```suggestion
        window?.rootViewController = navigationController()
```
이거 수정하시고 pull 후 다시 실행시켜보세요